### PR TITLE
Update Helm to enable ingress

### PR DESCRIPTION
**Pull request by LucasCairns**
- Create helpers to generate host names
- Split site and environement config in values files
- Update Ingress and Notes template to use helper functions

### CODE REVIEW

File: helm_deploy/prisoner-content-hub-frontend/values.cw.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+application:
+  config:
+    appName: "HMYOI Cookham Wood"
+    establishmentName: cookhamwood
+
+ingress:
+  domainPrefix: cw
```
Comment (spikeheap) [inline helm_deploy/prisoner-content-hub-frontend/values.cw.yaml:7]: Could we use `cookhamwood` (with or without hyphens) or something more descriptive in the name, domain, and references in `config.yml` here so the URL is meaningful?

File: .circleci/config.yml [hunk 2]
```diff
@@ -264,8 +261,7 @@ jobs:
     steps:
       - release_to_namespace:
           namespace: "prod"
-          establishment: "cookhamwood"
-          establishmentDisplayName: "HMYOI Cookham Wood"
+          establishment: "cw"
           releaseName: "prisoner-content-hub-frontend-cw"
```
Comment (spikeheap) [inline .circleci/config.yml:265]: These two lines could do with updating to `cookhamwood` for consistency, and we should rename `values.cw.yaml` to `values.cookhamwood.yaml`

File: helm_deploy/prisoner-content-hub-frontend/values.cw.yaml (hunk 1 above)
Comment (LucasCairns) [reply helm_deploy/prisoner-content-hub-frontend/values.cw.yaml]: Updated domain prefix

File: .circleci/config.yml (hunk 2 above)
Comment (LucasCairns) [reply .circleci/config.yml:265]: Updated Circle and filename

**Pull request merged**